yoti-sdk-api/src/main/java/com/yoti/api/client/docs/session/retrieve/configuration/capture/document/RequiredDocumentResourceResponse.java

```diff
@@ -0,0 +1,28 @@
+package com.yoti.api.client.docs.session.retrieve.configuration.capture.document;
```

Comment:
Unused imports again

I'm not sure I'd put these in their own _document_ package.  If we are going to do that, we should probably have subpackages for _liveness_ and _face_capture_ too